src/marqo/core/index_management/index_management.py

```diff
@@ -55,8 +55,27 @@ class IndexManagement:
         '''
     )
 
-    def __init__(self, vespa_client: VespaClient):
+    _DEPLOYMENT_LOCK_PATH = '/marqo__deployment_lock'
```

Comment:
Having all lock paths in one place makes it easier to ensure to clashes. So perhaps you could have a `lock_deployment` method somewhere in a class that will have one method per distributed lock we're using for all of Marqo